src/lib.rs

```diff
@@ -115,6 +115,8 @@ pub mod interpreter;
 pub mod miniscript;
 pub mod policy;
 pub mod psbt;
+#[allow(missing_docs)]
+pub mod ord;
```

Comment:
Would it be useful to you if we exported the `inscriptions` module in `ord` and made more types public so you can use it as a crate?

Reply:
yes, I think it's a good idea. but also not really for sapio unfortunately since we have like the entire rust-bitcoin ecosystem forked with patches.

this patch set though should be upstreamable!

Reply:
> Would it be useful to you if we exported the `inscriptions` module in `ord` and made more types public so you can use it as a crate?

I would love that.

Reply:
Have a look if this is useful: https://github.com/ordinals/ord/pull/3042